src/components/Blocking/ReportDomain.js

```diff
@@ -141,7 +142,7 @@ const styles = StyleSheet.create({
   btnSkipText: {
     fontFamily: fonts.inter[700],
     fontSize: 14,
-    color: '#000',
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the hard-coded color with a dynamic one from the theme object is a good practice, improving modularity and consistency.
```diff
- color: '#000',
+ color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->